service/src/main/java/io/camunda/service/ElementInstanceServices.java

```diff
@@ -147,6 +155,26 @@ private FlowNodeInstanceEntity toCacheEnrichedFlowNodeInstanceEntity(
         : item.withFlowNodeName(cachedItem.getElementName(item.flowNodeId()));
   }
 
+  public SearchQueryResult<IncidentEntity> searchIncidents(
+      final long elementInstanceKey, final IncidentQuery query) {
+    final IncidentQuery safeQuery = (query != null) ? query : IncidentQuery.of(b -> b);
+    final var authenticatedIncidentServices = incidentServices.withAuthentication(authentication);
+    if (authenticatedIncidentServices == null) {
+      return SearchQueryResult.of();
+    }
+    return authenticatedIncidentServices.search(
```

Comment:
💭 Should we make this consistent with the [process instance services](https://github.com/camunda/camunda/blob/09f1d0cf84f9df9d632df44c03d6754d158b308a/service/src/main/java/io/camunda/service/ProcessInstanceServices.java#L335)? Meaning, if we see a need to do those checks, we should also apply them there. If we don't, let's remove them here as well.